microsoft-dynamics-365-business-central/1.0/modules/ROOT/pages/microsoft-dynamics-365-business-central-connector-examples.adoc

```diff
@@ -0,0 +1,1564 @@
+                        <small id="numberNote" class="text-muted">At most 20 characters long</small><br>
+                        <label>Display Name</label>
+                        <input type="text" name="name" placeholder="My Item Name" maxlength=100 value="" required class="form-control">
+                        <small id="displayNameNote" class="text-muted">At most 100 characters long</small><br><br>
+                        <input class="btn btn-lg btn-primary btn-block" type="submit" value="Create"><br>
+                    </form>
+                    <div>Created ID: <div id="createdId"></div></div>
+                </div>
+            </div>
+        </div>
+
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h2 class="panel-title demo-heading">
+                    <a data-toggle="collapse" data-parent="#accordion" href="#operationFour">Update Item</a>
+                </h2>
+            </div>
+            <div id="operationFour" class="panel-collapse collapse">
+                <div class="panel-body">
+                    <form id="operationFourForm" class="demo-form" role="form" data-url="/item/update">
+                        <label>ID of item to update</label>
+                        <input type="text" name="id" placeholder="00000000-0000-0000-0000-000000000000" value="" required class="form-control"><br>
+                        <label>New Name</label>
+                        <input type="text" name="name" placeholder="New Item Name" value="" required class="form-control">
+                        <small id="displayNameNote" class="text-muted">At most 100 characters long</small><br><br>
+                        <input class="btn btn-lg btn-primary btn-block" type="submit" value="Update"><br>
+                    </form>
+                </div>
+            </div>
+        </div>
+
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h2 class="panel-title demo-heading">
+                    <a data-toggle="collapse" data-parent="#accordion" href="#operationFive">Delete Item</a>
+                </h2>
+            </div>
+            <div id="operationFive" class="panel-collapse collapse">
+                <div class="panel-body">
+                    <form id="operationFiveForm" class="demo-form" role="form" data-url="/item/delete">
+                        <label>ID of item to delete</label>
+                        <input type="text" name="id" placeholder="00000000-0000-0000-0000-000000000000" value="" required class="form-control"><br>
+                        <input class="btn btn-lg btn-primary btn-block" type="submit" value="Delete">
+                        <small id="deleteNote" class="text-muted">This action is irreversible.</small><br>
+                    </form>
+                </div>
+            </div>
+        </div>
+
+    </div>
+</div>
```

Comment:
The template contains all the operations in it, but always only one will work so we can simplify these templates I think, something like this should work:

```suggestion
<script type="text/javascript">
		$(document).ready( function() {

			var operationOneForm 	= $('#operationOneForm');

			handleSubmit = function(e) {
			  form = $(this);
              $.ajax({
                type: 'GET',							  	// Submit an HTTP POST request
                url: form.data('url'), 			      		// The URL where your endpoint is listening
                data: form.serialize(),	              		// Serialized form URL-encoded input
                success: function(data) {				  	// Success function called if request succeeds
                  alert("Operation Response : " + JSON.stringify(data));
                  if (form.data('ref-target')) {
                  	const fieldName = form.data('ref');
                  	const target = form.data('ref-target');
                  	$(target).html(data[fieldName]);
                  }
                },
                error: function(request, status, error){  // Error function is executed if an exception occurs in the flow
              	  alert(request.responseText);			  // Alert the user of any errors
              	}
              });

              return false;								  // Let jQuery handle the form submission
            };

			operationOneForm.submit(handleSubmit);
		});


</script>
<div class="container">
    <div class="panel-group" id="accordion">

        <div class="panel panel-default">
            <div class="panel-heading">
                <h2 class="panel-title demo-heading">
                    <a data-toggle="collapse" data-parent="#accordion" href="#operationOne">List Items</a>
                </h2>
            </div>
            <div id="operationOne" class="panel-collapse collapse in">
                <div class="panel-body">
                    <form id="operationOneForm" class="demo-form" role="form" data-url="/items">
                        <input class="btn btn-lg btn-primary btn-block" type="submit" value="List"><br>
                    </form>
                </div>
            </div>
        </div>

    </div>
</div>
```